### PR TITLE
move service-catalog prow config to specific directory

### PR DESCRIPTION
**Pull request by MHBauer**
/cc @BenTheElder @stevekuznetsov 

Should I also move the tide config to a `tide.yaml` ?  No, [see below](https://github.com/kubernetes/test-infra/pull/8718#issuecomment-406032507).

Doing this in preparation to add an e2e test

### CODE REVIEW

**Comment by krzyzacy** [comment]
name your file more specific, like `service-catalog-presubmits.yaml` since we use basename as a configmap key :-)

also add an OWNERS file and opt yourself in :-)

File: config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-incubator/service-catalog:
+  - name: pull-service-catalog-unit
+    agent: kubernetes
+    context: pull-service-catalog-unit
+    always_run: true
+    skip_report: true
```
[the remark this answers is not in the document]
Comment (MHBauer) [reply config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml]: oh shoot

[the remark this answers is not in the document]
Comment (MHBauer) [reply config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml]: no mergey. bad copy.

[the remark this answers is not in the document]
Comment (MHBauer) [reply config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml]: is skip_report=false the default? should I just remove the key?

[the remark this answers is not in the document]
Comment (stevekuznetsov) [reply config/jobs/kubernetes-incubator/service-catalog/service-catalog-presubmits.yaml]: Yep, `false` is the default (for all booleans, as with k8s api) and you can omit the key

**Comment by MHBauer** [comment]
Okay, I fixed it. Should be really ready this time.

**Pull request merged**